test/csharp/no-color-command-option-test/test.ts

```diff
@@ -22,13 +22,17 @@ describe("no-color option for cdktf deploy, diff, destroy", () => {
         cwd: driver.workingDirectory,
       }
     );
+    // These tests are sometimes flaky, therefore we log the result here to ensure we can debug it properly
+    console.log(result.stdout);
     expect(hasAnsi(result.stdout)).toBe(false);
   });
   onPosix("contains no color formatting in cdktf diff", async () => {
     const result = await execa("cdktf", ["diff", "--no-color"], {
       env: driver.env,
       cwd: driver.workingDirectory,
     });
+    // These tests are sometimes flaky, therefore we log the result here to ensure we can debug it properly
+    console.log(result.stdout);
```

Comment:
Are we sure that the console.logs would preserve the color of `result.stdout`? I guess if the bash color flags are coming through It should be fine– more of a question than concern 😁.

Reply:
Yeah, they did in past runs that failed we could see the color in these logs 👍